test_engine/.gitignore

```diff
@@ -0,0 +1,18 @@
+.tox
```

Comment:
Do we need this .gitignore? I think we have one able to reuse in the project folder.

Reply:
I have removed this file